framework/packages/abstract-core/src/registry.rs

```diff
@@ -19,10 +19,11 @@ pub const ICS20: &str = "ics-20";
 
 // chain-id prefixes based on `https://cosmos.directory/`
 pub const JUNO: &[&str] = &["juno", "uni"];
-pub const OSMOSIS: &[&str] = &["osmosis", "osmo"];
+pub const OSMOSIS: &[&str] = &["osmosis", "osmo", "osmo-test"];
```

Comment:
interesting, so "osmo" prefix doesn't cover "osmo-test"?

Reply:
No because of the way that it's "calculated" it just removes the string after the last hyphen.